docs/en/operation/Server-Logging.md

```diff
@@ -57,7 +59,7 @@ log4j.rootLogger=DEBUG, ${alluxio.logger.type}, ${alluxio.remote.logger.type}`
 ```
 
 ### Modifying Logging at Runtime
-
+TODO: this seems awfully familiar, similar to the analogous section in Command Line Interface???
```

Comment:
familiar to you, but probably not familiar to people reading this page. It's easier for the reader if important information is inlined instead of linked

Reply:
sure, this seems like information that should stay in this doc since it's titled "Server logging". i would like to link the CLI doc to this section, what say you?

Reply:
my issue is that they get out of sync; i made edits to the section in the CLI doc but not here.